pinot-plugins/pinot-batch-ingestion/pinot-batch-ingestion-standalone/src/main/java/org/apache/pinot/plugin/ingestion/batch/standalone/SegmentGenerationJobRunner.java

```diff
@@ -168,7 +173,6 @@ public void run()
     //Get list of files to process
     String[] files = _inputDirFS.listFiles(_inputDirURI, true);
 
-    //TODO: sort input files based on creation time
```

Comment:
Let's keep this TODO

Reply:
It's still there (with more details), I'd moved it up to before getting the file list, but moved it back down to reduce confusion.